charts/ocis/templates/nats/service.yaml

```diff
@@ -16,7 +16,9 @@ spec:
     - name: nats
       port: 9233
       protocol: TCP
+      appProtocol: tcp
```

Comment:
There is one missing.

Reply:
right, I wanted to leave this one on tcp but actually there needs to be a way to configure it to all-caps-tcp :man_facepalming: 

The question now is if we're offering a `nats: tcp` mapping as default? Or should we have a `tcp: tcp` mapping? (Also this only affects the builtin NATS, which shouldn't be used anyways :cry: )

Reply:
Similar to ldap that should be `nats: tcp`